ui/public/index.html

```diff
@@ -29,6 +29,39 @@
     -->
     <title>CERN Library Catalogue</title>
   </head>
+  <!-- Piwik -->
+  <script type="text/javascript">
+    var pkBaseURL =
+      'https:' == document.location.protocol
+        ? 'https://piwik.web.cern.ch/'
+        : 'http://piwik.web.cern.ch/';
+    document.write(
+      unescape(
+        "%3Cscript src='" +
+          pkBaseURL +
+          "piwik.js' type='text/javascript'%3E%3C/script%3E"
+      )
+    );
+  </script>
+  <script type="text/javascript">
+    try {
+      var piwikTracker = Piwik.getTracker(
+        pkBaseURL + 'piwik.php',
+        process.env.REACT_APP_INVENIO_PIWIK_ID
```

Comment:
you can test this locally, you can change the `.env` [here](https://github.com/CERNDocumentServer/cds-ils/blob/master/ui/.env) and it should be working.

Reply:
I tested it and it seems to be working (I didn't get any errors and the variable is set correctly)